test/models/maintenance_tasks/run_test.rb

```diff
@@ -164,6 +164,33 @@ class RunTest < ActiveSupport::TestCase
       run.persist_error(error)
     end
 
+    test "#persist_error does not raise on longer error class names" do
+      run = Run.create!(task_name: "Maintenance::ErrorTask")
+      error = ArgumentError.new("Something went wrong")
+      class_name = "SomeVeryLongErrorClassName #{"." * 20000}"
+      error.stubs(class: class_name)
```

Comment:
(This is weird - `class` should return a Class object, not a string. Can we instead create a new error class with a name that's > 255 character long?)